### PR TITLE
wolfictl update package... new CLI to update a melange config + automate secfixes with git commit comments

### PR DESCRIPTION
**Pull request by rawlingsj**
Screenshot 2023-02-01 at 22 35 26

Example:

```
wolfictl update package cosign --version v2.1.3cg2 --target-repo https://github.com/wolfi-dev/os
```

This will create a pull request on the target repo containing the melange config that needs to be updated.

This is expected to be run from a release pipeline repo that contains the source and fixes to be included in the updated melange apk.  

If there are commits from the source repo that are part of a new release and contain commit message 
```
fixes: CVE###
```
then the PR to update the melange config will auto generate the advisory and secfixes.

tl;dr if you fix a CVE include `fixes: CVE###` and when the automated release pipeline updates the downstream melange config via PR it will create a "fixed" advisory.

### CODE REVIEW

**Comment by rawlingsj** [comment]
Thanks to @imjasonh for the idea to link fixes to generate the advisories!

**Review by jdolitsky — APPROVED** [review]
